### DISCUSSION TITLE
çözünürlük seçme

### DISCUSSION THREAD

**Discussion by ahmehmet** [KebabLord/turkanime-indirici#39] [Genel] [answered]
Biliyorum artık biraz fazla oluyorum ama eğer mümkün değil veya çok zorsa nedenini öğrenmek istedim. 
İzlerken veya indirirken yalnızca 720p seçeneğimiz olduğunu fark ettim. Mesela bölüm seçtikten sonra ya da direkt ayarlar kısmından 1080p seçeneğini önceleme ayarı eklenemez mi? Yani eğer içinde 1080p olan ve yaşayan bir player varsa önce ondan idirmeyi/izletmeyi başlatmasını sağlayabilir miyiz?

**Comment by KebabLord** [comment]
youtube-dl pek tabi çözünürlükler arasında seçim yapma kabiliyetine sahip ve bunu entegre etmek mümkün. Uygulamanın önceki versiyonlarında ilkel bir çözünürlük ayarı da vardı. Tek sorun her bir fansub içerisindeki her bir player alternatifini tek tek çözünürlük adına kontrol etmek programı hatrı sayılır ölçüde yavaşlatır gibi. Yine de elimizdeki videoyu en yüksek çözünürlükte indirmeye yönelik bir ayar eklenebilir sanırım

**Comment by KebabLord** [comment, marked as answer]
Başlıkta belirtilen özellik [v8.2.2](https://github.com/KebabLord/turkanime-indirici/tree/v8.2.2) sürümünden itibaren 2 yıl önce uygulamaya eklendi. Script başarıyla 1080p'yi bulabiliyor.
